rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -28,12 +30,23 @@ include::rpa-home::partial$windows-versions-support.adoc[]
 
 |===
 
-=== Requirements for Secure Session Technology
+=== Requirements for the Secure Session Technology
+
+Secure Sessions are hidden Desktop sessions. They run in the background and only the bot can access them. An additional advantage of using Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
 
-* Windows logins used for secure sessions have privileges to open a local RDP session on `127.0.0.2`.
-* 64-bit Operating System.
-* Dedicated Windows Server machine with enough resources to execute several processes simultaneously (2 GB or more RAM per session)
-* The RPA Bot service runs under the _Local System account_ (by default, the setup configures RPA Bot to use this account).
+* The RPA Bot service runs under the _Local System_ account per default.
+// question: correct requirements?
+** If you want to use another account, ensure that it has permission to run Windows services.
+// question: local account vs domain account?
+* The RPA Bot service runs processes under user accounts existing on the same computer or within the same domain.
+// question: Is it enough that the bot users belong to the _Remote Desktop Users_ group or is only the port 127.0.0.2 allowed? Do we need to mention this port specificially?
+** We recommend using dedicated accounts and a user naming scheme to differentiate between local users and domain users and between human users and bot users.
+** User accounts used by the bot to run processes must belong to the _Remote Desktop Users_ group and must be allowed to establish a local RDP session via the port 127.0.0.2.
```

Comment:
Instead of 'port' I would prefer the word 'address' or the term 'local loopback address'